[2025-03-30 22:19:21,876] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: natality_web_data_to_gcs.download_race_lookup manual__2025-03-30T22:19:16.133079+00:00 [queued]>
[2025-03-30 22:19:21,893] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: natality_web_data_to_gcs.download_race_lookup manual__2025-03-30T22:19:16.133079+00:00 [queued]>
[2025-03-30 22:19:21,893] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2025-03-30 22:19:21,894] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2025-03-30 22:19:21,894] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2025-03-30 22:19:21,914] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_race_lookup> on 2025-03-30 22:19:16.133079+00:00
[2025-03-30 22:19:21,936] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'natality_web_data_to_gcs', 'download_race_lookup', 'manual__2025-03-30T22:19:16.133079+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/bigquery_to_gcs.py', '--cfg-path', '/tmp/tmphczbkrfs', '--error-file', '/tmp/tmppf1c53xb']
[2025-03-30 22:19:21,927] {standard_task_runner.py:52} INFO - Started process 230 to run task
[2025-03-30 22:19:21,945] {standard_task_runner.py:77} INFO - Job 78: Subtask download_race_lookup
[2025-03-30 22:19:22,102] {logging_mixin.py:109} INFO - Running <TaskInstance: natality_web_data_to_gcs.download_race_lookup manual__2025-03-30T22:19:16.133079+00:00 [running]> on host 6e74f7e5ce4c
[2025-03-30 22:19:22,207] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=natality_web_data_to_gcs
AIRFLOW_CTX_TASK_ID=download_race_lookup
AIRFLOW_CTX_EXECUTION_DATE=2025-03-30T22:19:16.133079+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-30T22:19:16.133079+00:00
[2025-03-30 22:19:22,212] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-03-30 22:19:22,213] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'wget --no-check-certificate "https://drive.google.com/file/d/17o8e1eh7XkHk61P5JuBAi2JhFvnUt4tJ/view?usp=drive_link" -O /tmp/race_lookup.csv']
[2025-03-30 22:19:22,235] {subprocess.py:85} INFO - Output:
[2025-03-30 22:19:22,269] {subprocess.py:89} INFO - --2025-03-30 22:19:22--  https://drive.google.com/file/d/17o8e1eh7XkHk61P5JuBAi2JhFvnUt4tJ/view?usp=drive_link
[2025-03-30 22:19:22,298] {subprocess.py:89} INFO - Resolving drive.google.com (drive.google.com)... 173.194.193.101, 173.194.193.138, 173.194.193.113, ...
[2025-03-30 22:19:22,300] {subprocess.py:89} INFO - Connecting to drive.google.com (drive.google.com)|173.194.193.101|:443... connected.
[2025-03-30 22:19:22,631] {subprocess.py:89} INFO - HTTP request sent, awaiting response... 200 OK
[2025-03-30 22:19:22,631] {subprocess.py:89} INFO - Length: unspecified [text/html]
[2025-03-30 22:19:22,631] {subprocess.py:89} INFO - Saving to: ‘/tmp/race_lookup.csv’
[2025-03-30 22:19:22,632] {subprocess.py:89} INFO - 
[2025-03-30 22:19:22,635] {subprocess.py:89} INFO -      0K .......... .......... .......... .......... .......... 13.7M
[2025-03-30 22:19:22,639] {subprocess.py:89} INFO -     50K .......... .......... .......... ..........            13.3M=0.007s
[2025-03-30 22:19:22,639] {subprocess.py:89} INFO - 
[2025-03-30 22:19:22,640] {subprocess.py:89} INFO - 2025-03-30 22:19:22 (13.5 MB/s) - ‘/tmp/race_lookup.csv’ saved [92948]
[2025-03-30 22:19:22,640] {subprocess.py:89} INFO - 
[2025-03-30 22:19:22,640] {subprocess.py:93} INFO - Command exited with return code 0
[2025-03-30 22:19:22,707] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=natality_web_data_to_gcs, task_id=download_race_lookup, execution_date=20250330T221916, start_date=20250330T221921, end_date=20250330T221922
[2025-03-30 22:19:22,769] {local_task_job.py:154} INFO - Task exited with return code 0
[2025-03-30 22:19:22,834] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
